Create devjourney.mdx (#90)
Swizec/swizec.com: src/pages/interviews/devjourney.mdx

[DevJourney Podcast Episode #202](https://devjourney.info/Guests/202-SwizecTeller.html)

## Talking points

- Swizec's journey into programming
- Running out of variables in Pascal
- Swizec got in trouble with Paul Graham and Ycombinator and nearly got his startup kicked out.
- the biggest mindset shifts that helped me go from code monkey to tech lead* Moving to Silicon Valey
- Contributing code to a nuclear power plant
- Working for free on open-source vs working for money
- The sad story of CURL, the command line utility
- Innovation and entrepreneurship
- The biggest mistake we made in startup fundraising
- How Swizec did an intersnhip in Palo Alto and ended up living next door to Paul Graham.
- You should be in the Silicon Valley ecosystem if you want to make it really big

## Quotable Quotes

"you know how some people, some kids get really obsessed with, uh, action figures and action heroes and stuff. For me, that was programming" - ST

> "I was in my twenties and I didn't really realize that programming concepts actually had meanings in reality. LIN means 'write line' because you're writing out a lin.I remember waking up and like, oh my God, these things have actual meanings. That's amazing. It's makes it so much easier." - ST

Reviewer: ```suggestion
"I was in my twenties and I didn't really realize that programming concepts actually had meanings in reality. `WriteLn` means 'write line' because you're writing out a line. I remember waking up and like, oh my God, these things have actual meanings. That's amazing. It's makes it so much easier." - ST
```

"I want to spend all my time programming and writing cool stuff" - ST

> "I didn't know, you have to name variable. So when I ran out of 25 variables, a a through Z, I started doing AA a, B a. I think I basically forced myself to do assembly style programming just in terrible pascals." - ST

Reviewer: ```suggestion
"I didn't know, you have to name variable. So when I ran out of 25 variables, A through Z, I started doing AA, BB. I think I basically forced myself to do assembly style programming just in terrible pascals." - ST
```

"Mostly never build your own framework" - ST

"Everyone else who comes after him suffers because there's no public documentation, there's nobody you can ask for help. It's just that guy and his pet project and anyone else at the company who knows." - ST

"Nobody cares what technology you're using. What you do with that technology is what matters." - ST

"Silicon valley minded people, are looking for someone who can come in and push the business to the next level, rather than being a code monkey who can take orders and execute with technology" - ST

> "As soon as you feel like you're doing the same shit over and over, just switch jobs, go to a different companies" - ST

Reviewer: ```suggestion
"As soon as you feel like you're doing the same shit over and over, just switch jobs, go to different companies" - ST
```

> "Start something of your own, starting your own is a great way to learn the very early stages and to see into how the people who are hiring you think about these things and then sell yourself based on that." - ST

Reviewer: ```suggestion
"Start something of your own – starting your own is a great way to learn the very early stages and to see into how the people who are hiring you think about these things and then sell yourself based on that." - ST
```

> "Now is the time to push. I'll have time to chill later" - ST
> "don't push too much, so you don't enjoy life, but that's a different thing." - ST

Reviewer: ```suggestion
"Now is the time to push. I'll have time to chill later ... but don't push too much, so you don't enjoy life" - ST
```

"If you want to move towards Silicon valley. I think the best thing is to start working with tier two and tier three companies that you can find locally, get that experience in a high pace, high growth environment" - ST

"I've heard stories of people being hired straight out of college as senior software engineer, because there's been a bit of title inflation." - ST

"I learned people actually can pay you millions of dollars. Or you can do it for free and suffer" - ST

"CURL is maintained by a single guy who repeatedly runs out of money and goes online, asking for donations, because he's like, yo, I'm about to get evicted. I can't afford rent." - ST

"I went to this museum and there was a really good exhibit on all of the scientists that changed the world. And I realized, oh my God, they weren't just scientists. These people were actually entrepreneurs. The science was just a small part." - ST

"James watt didn't invent the steam engine. He took a steam engine that existed, invented a small part that made it actually useful and then capitalized on that event." - ST 

> "Thomas Edison didn't kind of invented the light bulb, but his actual innovation was the research lab, the commercial research lab" - ST

Reviewer: ```suggestion
"Thomas Edison kind of invented the light bulb, but his actual innovation was the research lab, the commercial research lab" - ST
```

"We had a scarcity mindset, not a abundance mindset" -ST

"We had this VC saying, oh yeah, you, we will give you money. If you get more people involved. And our reaction to that was, well, we can't afford to stay. So we're going to go home because we, we have no money and we are poor." - ST

"I took full responsibility for what I did, because you know, that's what you do, ownership, mentality and all that" - ST

"In 2021, there was around a hundred billion dollars of startup investment in the bay area ecosystem." - ST

"If you think of VCs as the big studios, CEOs and founders, as the directors, us engineers, et cetera, as the talent. Absolutely anywhere in the world, you can make a really great movie. You might even win an Oscar, like really good movies come from everywhere. But if you want to make a blockbuster, you have to go to Hollywood, maybe not physically to Hollywood, but you have to be in the ecosystem." - ST

"The way you can recognize the tier three company is that they compete for global talent." - ST

> "Once they get big enough, they only want to top tier talent and they treat people as an investment, not as a cost like an engineer at one of these companies is an investment" - ST

Reviewer: ```suggestion
"Once they get big enough, they only want to top tier talent and they treat people as an investment, not as a cost" - ST
```

"If you want to work with companies where people who know how to take, to take, to take a business from 10 employees to 50 employees from 50 to 300, from 300 to a thousand, from a thousand to 10,000, those people are. Are a dime a dozen in the bay area." - ST

"There's a lot of people in SF who know what that process looks like from all aspects of the company, not just from leadership, but also from engineering, because the way you approach engineering when your company is growing on that trajectory is very different than the way you approach engineering." - ST

> "Even if business opportunities like this existed in Europe, you don't have the talent who knows how to pull this off. If you're a founder and you have, and you're like, I'm on a trajectory to go from $10 million to a hundred million dollars in a year and a half. Good luck finding people who know how that works at all levels of the organization." - ST

Reviewer: ```suggestion
"Even if business opportunities like this existed in Europe, you don't have the talent who knows how to pull this off. If you're a founder and you're like, I'm on a trajectory to go from $10 million to a hundred million dollars in a year and a half. Good luck finding people who know how that works at all levels of the organization." - ST
```

> "It's like you have less volatility. You're not going to become a trillionaire. Probably not even a billionaire. Honestly, even if I'm a dozen millionaire, I would be totally fine, probably anywhere in the world. You know, you don't need to be a billionaire, even if you just have a couple of million dollars, it's fine." - ST

Reviewer: ```suggestion
"It's like you have less volatility. You're not going to become a trillionaire. Probably not even a billionaire. Honestly, even if I'm a dozen millionaire, I would be totally fine anywhere in the world." - ST
```

<iframe style="border-radius:12px" src="https://open.spotify.com/embed/episode/36cOMQZCkjsTE8baZt82VR?utm_source=generator&t=0" width="100%" height="232" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>

[DevJourney Podcast Episode #202](https://devjourney.info/Guests/202-SwizecTeller.html)

## Talking points

- Swizec's journey into programming
- Running out of variables in Pascal
- Swizec got in trouble with Paul Graham and Ycombinator and nearly got his startup kicked out.
- the biggest mindset shifts that helped me go from code monkey to tech lead* Moving to Silicon Valey
- Contributing code to a nuclear power plant
- Working for free on open-source vs working for money
- The sad story of CURL, the command line utility
- Innovation and entrepreneurship
- The biggest mistake we made in startup fundraising

> - How Swizec did an intersnhip in Palo Alto and ended up living next door to Paul Graham.

Reviewer: ```suggestion
- How Swizec did an internship in Palo Alto and ended up living next door to Paul Graham.
```
